frontend/build-configs/bayern/index.ts

```diff
@@ -3,6 +3,7 @@ import publisherText from "./publisherText"
 import disclaimerText from "./disclaimerText"
 
 export const bayernCommon: CommonBuildConfigType = {
+    activationPath: '/activation',
```

Comment:
What's this used for? When is it not '/activation'?

Reply:
we can use this path for the router in the native frontend (modal will open to activate card). Furthermore it is used for the ios universal links to whitelist certain paths that open the app (apple-app-site-association)